source/_build/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
```

Comment:
I don't think the ``_build`` folder should be added to git (in the same way that the contents of ``documentation`` is currently not